### PR TITLE
open source

### PR DESCRIPTION
**Pull request by jsfez**
- feat: improve navbar
- feat: improve slogan
- feat: add open source section

### CODE REVIEW

File: containers/AppNavbar.tsx [hunk 1]
```diff
@@ -14,15 +16,23 @@ export const AppNavbar: React.FC = () => {
...
           <NavbarLink href="https://argos-ci.com/docs/">Docs</NavbarLink>
           <NavbarLink href="/pricing">Pricing</NavbarLink>
-          <NavbarLink href="/blog">Blog</NavbarLink>
-          <NavbarLink href="https://app.argos-ci.com/login">Login</NavbarLink>
+          <NavbarLink href="https://github.com/argos-ci/argos">
+            <div className="flex gap-1 items-center">
+              <GithubLogo className="w-4 h-4 shrink-0" />
+              GitHub
+            </div>
+          </NavbarLink>
```
Comment (gregberge) [inline containers/AppNavbar.tsx:27]: I think GitHub should not be in the top navbar. It is secondary, going in GitHub is useless. So we should remove it, footer is enough.

File: pages/index.tsx [hunk 2]
```diff
@@ -45,27 +46,24 @@ export default function Home() {
                 Playwright support
               </a>
             </Chip>
-            <h1 className="text-4xl sm:leading-tight sm:text-6xl font-bold bg-clip-text">
+            <h1 className="text-4xl sm:leading-tight sm:text-6xl font-bold bg-clip-text max-w-2xl mx-auto">
               <span className="bg-clip-text text-transparent bg-gradient-to-b from-white to-[#AAAAAA]">
-                Detect bugs
```
Comment (gregberge) [inline pages/index.tsx:50]: If you change it you have to change it everywhere:

- Marketplace
- Product Hunt
- Twitter
- etc...

- Social image

It is like a "rebranding".

**Review by gregberge — CHANGES_REQUESTED** [review]
I think most of changes are not concerning the open source container on the pricing page.

You should only change the pricing page and creates tickets for other things.

- Brand remastering
- GitHub in navbar (no)

File: containers/AppNavbar.tsx (hunk 1 above)
Comment (jsfez) [reply containers/AppNavbar.tsx:27]: I understand your point, but I see the GitHub link in the header as a reflection of our identity as developers. It's more about our brand image than its actual utility. What are your thoughts?

Comment (gregberge) [reply containers/AppNavbar.tsx:27]: Then you should probably link to Organization. I think the best would be a page explicating that we are open source instead.

File: pages/index.tsx (hunk 2 above)
Comment (jsfez) [reply pages/index.tsx:50]: We seem to have different slogans across our platforms. I'm opening a ticket to unify them and keep our messaging consistent.

Comment (jsfez) [reply pages/index.tsx:50]: https://github.com/orgs/argos-ci/projects/1/views/1?pane=issue&itemId=32807419

**Pull request closed without merging**